test/main/store/migrations/index.test.js

```diff
@@ -931,3 +931,55 @@ describe('migration 25', () => {
     })
   })
 })
+
+describe('migration 26', () => {
+  beforeEach(() => {
+    state = {
+      main: {
+        _version: 20,
+        networks: {
+          ethereum: { 
+            5: {
+              id: 5,
+              type: 'ethereum',
+              layer: 'testnet',
+              symbol: 'ETH',
+              name: 'Görli',
+              explorer: 'https://goerli.etherscan.io',
+              gas: {
+                price: {
+                  selected: 'standard',
+                  levels: { slow: '', standard: '', fast: '', asap: '', custom: '' }
+                }
+              },
+              connection: {
+                primary: { on: true, current: 'infura', status: 'loading', connected: false, type: '', network: '', custom: '' },
+                secondary: { on: false, current: 'custom', status: 'loading', connected: false, type: '', network: '', custom: '' }
+              },
+              on: false
+            },
+            10: {
+              connection: {
+                primary: { current: 'custom' },
+                secondary: { current: 'local' }
+              }
+            }
+          }
+        },
+        networksMeta: {
+          ethereum: { }
+        },
+        networkPresets: {
+          ethereum: { }
+        }
+      }
+    }
+  })
+
+  it('removes the symbol property on a network', () => {
+    const updatedState = migrations.apply(state, 26)
+    const networks = updatedState.main.networks.ethereum
+    
+    expect(Object.hasOwn(networks[5], "symbol")).toBeFalsy()
```

Comment:
for tests we usually want to test what an object looks like to the application, not how its properties are implemented. in this case, we want `chain.symbol` to be falsy so we can just explicitly check for that